patches/src/main/kotlin/app/revanced/patches/angulus/RemoveAdsPatch.kt

```diff
@@ -0,0 +1,18 @@
+package app.revanced.patches.angulus
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+@Suppress("unused")
+val angulusPatch = bytecodePatch(
+    name = "Hide ads",
+    description = "Hides in app ads",
+) {
```

Comment:
A description is not necessary here.

```suggestion
val angulusPatch = bytecodePatch(name = "Hide ads") {
```